Scripts/Core/EditorVR.Interfaces.cs

```diff
@@ -51,12 +51,22 @@ internal void ConnectInterfaces(object obj, Transform rayOrigin = null)
 					connectInterfaces(obj, rayOrigin);
 			}
 
-			internal void DisconnectInterfaces(object obj)
+			internal void DisconnectInterfaces(object obj, InputDevice device)
```

Comment:
Why this? Looking at how you're calling it you could have just passed the deviceData and since you could have passed that, then why not simply pass the rayOrigin?